### PR TITLE
65315_authenticate_social_network

### PR DESCRIPTION
**Pull request by asifbuetcse**

### CODE REVIEW

File: app/src/Controller/AppController.php [hunk 1]
```diff
@@ -69,6 +69,17 @@ public function initialize()
                 'action' => 'index'
             ]
         ]);
+
+        $this->loadComponent('AkkaFacebook.Graph', [
+            'app_id' => '1803823816512864',
```
Comment (hieubm) [inline app/src/Controller/AppController.php]: can you make it configurable by environment variables ?

File: app/src/Controller/AppController.php [hunk 2]
```diff
@@ -69,6 +70,17 @@ public function initialize()
                 'action' => 'index'
             ]
         ]);
+
+        $this->loadComponent('AkkaFacebook.Graph', [
+            'app_id' => Configure::read('facebookConfiguration.appId'),
+            'app_secret' => Configure::read('facebookConfiguration.appSecret'),
+            'app_scope' => Configure::read('facebookConfiguration.appScope'),
+            'redirect_url' => 'http://localhost/fdf_2/app/users/login',
```
Comment (tuandv2611) [inline app/src/Controller/AppController.php]: You need make config for "redirect_url"

File: app/config/custom_settings.php [hunk 3]
```diff
@@ -2,6 +2,11 @@
 $config = [
     'fileUpload' => [
         'imageFilePath' => WWW_ROOT . 'img/',
+    ],
+    'facebookConfiguration' => [
+    	'appId' => '1803823816512864',
+    	'appSecret' => 'd3740f0575078760627161fa67f4888a',
+        'appScope' => 'email,public_profile',
```
Comment (tuandv2611) [inline app/config/custom_settings.php]: Indent

File: app/config/custom_settings.php [hunk 4]
```diff
@@ -2,6 +2,12 @@
 $config = [
     'fileUpload' => [
         'imageFilePath' => WWW_ROOT . 'img/',
+    ],
+    'facebookConfiguration' => [
+    	'appId' => '1803823816512864',
+    	'appSecret' => 'd3740f0575078760627161fa67f4888a',
+    	'appScope' => 'email,public_profile',
+    	'redirectUrl' => 'http://localhost/fdf_2/app/users/login',
```
Comment (hieubm) [inline app/config/custom_settings.php]: these settings should be stored in config/app.php
because, app.php will not be pushed to Git

**Pull request merged**